shuup/core/models/_products.py

```diff
@@ -143,10 +155,7 @@ def _visible(self, shop, customer, language=None):
                         shop_products__shop=shop,
                         shop_products__visibility=ShopProductVisibility.NOT_VISIBLE
                     )).filter(
-                mode__in=(
-                    ProductMode.NORMAL, ProductMode.PACKAGE_PARENT,
-                    ProductMode.SIMPLE_VARIATION_PARENT, ProductMode.VARIABLE_VARIATION_PARENT
-                )
+                mode__in=self._get_visible_modes()
```

Comment:
This commit should be squashed with the one which add SUBSCRIPTION product mode, since this adds the SUBSCRIPTION to this list.